qemu/tests/balloon_check.py

```diff
@@ -352,6 +360,9 @@ def run(test, params, env):
     """
     if params['os_type'] == 'windows':
         balloon_test = BallooningTestWin(test, params, env)
+        logging.info("Waiting 90s for guest's applications up")
+        time.sleep(90)
```

Comment:
Windows guest get its used memory when it connects to serial console, but at that time, the used memory is not stable, it always become greater, so set a sleep time to wait the used memory to become stable.